eos/rare-b-decays/observables.cc

```diff
@@ -1337,76 +1348,160 @@ namespace eos
                                   / <<B->K^*ll::Gamma>>
                         )"),
 
+                make_expression_observable("B->K^*ll::S_1s(q2)", R"(S_{1s}^{\bar{B}\to \bar{K}^*\ell^+\ell^-}(q^2))",
+                        Unit::None(),
+                        R"(
+                        2.0 / 3.0 * (<<B->K^*ll::J_1s(q2);cp-conjugate=false>> + <<B->K^*ll::J_1s(q2);cp-conjugate=true>>)
+                                  / <<B->K^*ll::Gamma(q2)>>
+                        )"),
+
                 make_expression_observable("B->K^*ll::S_1c", R"(S_{1c}(\bar{B}\to \bar{K}^*\ell^+\ell^-))",
                         Unit::None(),
                         R"(
                         2.0 / 3.0 * (<<B->K^*ll::J_1c;cp-conjugate=false>> + <<B->K^*ll::J_1c;cp-conjugate=true>>)
                                   / <<B->K^*ll::Gamma>>
                         )"),
 
+                make_expression_observable("B->K^*ll::S_1c(q2)", R"(S_{1c}^{\bar{B}\to \bar{K}^*\ell^+\ell^-}(q^2))",
+                        Unit::None(),
+                        R"(
+                        2.0 / 3.0 * (<<B->K^*ll::J_1c(q2);cp-conjugate=false>> + <<B->K^*ll::J_1c(q2);cp-conjugate=true>>)
+                                  / <<B->K^*ll::Gamma(q2)>>
+                        )"),
+
                 make_expression_observable("B->K^*ll::S_2s", R"(S_{2s}(\bar{B}\to \bar{K}^*\ell^+\ell^-))",
                         Unit::None(),
                         R"(
                         2.0 / 3.0 * (<<B->K^*ll::J_2s;cp-conjugate=false>> + <<B->K^*ll::J_2s;cp-conjugate=true>>)
                                   / <<B->K^*ll::Gamma>>
                         )"),
 
+                make_expression_observable("B->K^*ll::S_2s(q2)", R"(S_{2s}^{\bar{B}\to \bar{K}^*\ell^+\ell^-}(q^2))",
+                        Unit::None(),
+                        R"(
+                        2.0 / 3.0 * (<<B->K^*ll::J_2s(q2);cp-conjugate=false>> + <<B->K^*ll::J_2s(q2);cp-conjugate=true>>)
+                                  / <<B->K^*ll::Gamma(q2)>>
+                        )"),
+
                 make_expression_observable("B->K^*ll::S_2c", R"(S_{2c}(\bar{B}\to \bar{K}^*\ell^+\ell^-))",
                         Unit::None(),
                         R"(
                         2.0 / 3.0 * (<<B->K^*ll::J_2c;cp-conjugate=false>> + <<B->K^*ll::J_2c;cp-conjugate=true>>)
                                   / <<B->K^*ll::Gamma>>
                         )"),
 
+                make_expression_observable("B->K^*ll::S_2c(q2)", R"(S_{2c}^{\bar{B}\to \bar{K}^*\ell^+\ell^-}(q^2))",
+                        Unit::None(),
+                        R"(
+                        2.0 / 3.0 * (<<B->K^*ll::J_2c(q2);cp-conjugate=false>> + <<B->K^*ll::J_2c(q2);cp-conjugate=true>>)
+                                  / <<B->K^*ll::Gamma(q2)>>
+                        )"),
+
+                make_expression_observable("B->K^*ll::S_3(q2)", R"(S_3^{\bar{B}\to \bar{K}^*\ell^+\ell^-}(q^2))",
```

Comment:
Why is ``S_3(q2)`` barging the queue here? The others are **after** their respective integrated quantities.